TimJohnson/Homework_Week1.js

```diff
@@ -0,0 +1,139 @@
+1a)	"1" == 1; 	true;  using == means equal to and will cast the 1 on the right side of the equation as a string
+1b)	"1" === 1;  false;  using === means that the comparison includes not only equal value but equal type.  (here is string versus number)
+1c)	 x == 'x';	false;  string 'x' is converted tonumber(x) which is different than the number x
+1d)	 x == (x+''); true;  number is equal to that number + an empty string
+1e)	'' == ' ';	false;  empty sting is always false but ' ' is a valid string
```

Comment:
In this case we are not evaluating the validity of the strings, but rather whether the strings are equivalent to each other. Though it is false, it is because an empty string is not the same as a string containing a space.